gradle-mvn-push.gradle

```diff
@@ -45,11 +45,15 @@ def getRepositoryPassword() {
   return hasProperty('SONATYPE_NEXUS_PASSWORD') ? SONATYPE_NEXUS_PASSWORD : ""
 }
 
+def needsSigning() {
+  return isReleaseBuild() && gradle.taskGraph.hasTask("uploadArchives")
```

Comment:
Single-quote `'uploadArchives'`, since we do not need `$` expansion in this string.